src/constants/tokens.ts

```diff
@@ -91,9 +92,10 @@ export const TheUSDCYieldIndex: Token = {
 
 // Other
 
-export const ETH: Token = {
+export const ETH = {
   symbol: 'ETH',
   address: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+  decimals: 18,
 }
```

Comment:
Will start using this const more in tests to be able to get rid of `QuoteTokens` at some point. So once tokenlists is fully used this will probably be the only remaining const in here.